alkira/resource_alkira_group_direct_inter_connector.go

```diff
@@ -51,23 +52,29 @@ func resourceAlkiraDirectInterConnectorGroup() *schema.Resource {
 			"cxp": {
 				Description: "The CXP of the group.",
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
 			},
 			"connector_provider_region": {
 				Description: "The region of the connector.",
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
 			},
 			"connector_type": {
-				Description: "The type of the connector.",
-				Type:        schema.TypeString,
-				Required:    true,
+				Description:  "The type of the connector.",
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice([]string{"AWS_VPC", "AZURE_VNET"}, false),
 			},
 			"provision_state": {
 				Description: "The provisioning state of the resource.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
+			"virtual_network_manager_azure_id": {
+				Description: "The Azure Virtual Network Manager's Alkira ID.",
```

Comment:
The description is mis-leading. I think we should just say:

"ID of `virtual_network_manager_azure`.

Reply:
Basically we create a Virtual Network Manager on Azure and they have their own ID, that's why I explicitly mentioned Alkira ID.